### PR TITLE
Lab5.py

### PR DESCRIPTION
**Pull request by VitaliiPaliukh**

### CODE REVIEW

**Review by nazartsap21 — APPROVED** [review]

File: lab5weather.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        return result   
+         
+    def get_day(self, record):
+        return record.day
+
+    def sort_weather_info(self):
+        self.weather_info.sort(key = self.get_day)
+
+
+if __name__ == "__main__":
```
Comment (nazartsap21) [inline lab5weather.py]: should use separate file for main and prints

File: README.md [hunk 2]
```diff
@@ -1 +1,2 @@
-# Labspython.IR2023
\ No newline at end of file
+# Labspython.IR2023
+Написати програму алгоритмічною мовою Python згідно з завданням, отриманим від викладача за табл. 1. Додайте відповідні конструктори, деструктори, функції доступу та виведення інформації на екран. У методі main продемонструйте роботу класів.
```
Comment (nazartsap21) [inline README.md]: add full description of lab5

**Review by nazartsap21 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (VitaliiPaliukh) [reply README.md:2]: done

Comment (VitaliiPaliukh) [reply README.md]: fixed

File: lab5weather.py (hunk 1 above)
Comment (VitaliiPaliukh) [reply lab5weather.py]: separated

**Review by VitaliiPaliukh — COMMENTED** [review]
fixed

**Review by nazartsap21 — APPROVED** [review]

File: seperate main from lab5 [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+    print("\nWeather Records Sorted by Day:")
+    for record in calendar.weather_info:
+        print(record)
+
+    max_temp = calendar.find_max_temperature("2023-11-03")
+    print(f"\nMax Temperature for 2023-10-25: {max_temp}°C")
+
+    prediction = calendar.is_lviv_weather(92, WeatherType.RAINY)
+    print("\nLviv Weather Prediction:", prediction)
```
Comment (zveres-ss) [inline seperate main from lab5:29]: file is missing extention .py and name does not correspond to the standards

**Review by zveres-ss — CHANGES_REQUESTED** [review]

**Pull request merged**